src/Event/ApplicationExceptionEvent.php

```diff
@@ -26,7 +26,11 @@
 	private $exception;
 
 
-	public function __construct(Application $application, Exception $exception = NULL)
+	/**
+	 * @param Application $application
+	 * @param \Throwable|Exception|NULL $exception
```

Comment:
Could you unify it to `\Throwable|\Exception|NULL` and remove the use statement? Also don't forget the return annotation below.